org.testeditor.web.backend.persistence/src/test/java/org/testeditor/web/backend/persistence/TestExecutorTest.xtend

```diff
@@ -0,0 +1,129 @@
+package org.testeditor.web.backend.persistence
+
+import ch.qos.logback.classic.Level
+import ch.qos.logback.classic.Logger
+import ch.qos.logback.classic.spi.ILoggingEvent
+import ch.qos.logback.classic.spi.LoggingEvent
+import ch.qos.logback.core.Appender
+import java.util.regex.Pattern
+import org.assertj.core.api.Condition
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TemporaryFolder
+import org.mockito.ArgumentCaptor
+import org.mockito.Captor
+import org.mockito.InjectMocks
+import org.mockito.Mock
+import org.mockito.Mockito
+import org.mockito.MockitoAnnotations
+import org.slf4j.LoggerFactory
+import org.testeditor.web.backend.persistence.workspace.WorkspaceProvider
+
+import static org.assertj.core.api.Assertions.*
+import static org.mockito.Mockito.*
+
+class TestExecutorTest {
+
+	@Rule
+	public val temporaryFolder = new TemporaryFolder
+
+	@InjectMocks
+	TestExecutorProvider testExecutorProviderUnderTest
+
+	@Mock
+	WorkspaceProvider workspaceProviderMock
+	@Mock
```

Comment:
If we have many annotated fields I personally prefer having them on the same line. Alternatively, we should be consistent with the blank lines between the fields.